cfy_cluster_manager/main.py

```diff
@@ -831,7 +834,11 @@ def _remove_cloudify_installation(instance, verbose):
             if instance.file_exists(full_path):
                 new_path = join(certs_dir_path, timestamp + cert_path)
                 instance.run_command(
-                    'mv {0} {1}'.format(full_path, new_path), use_sudo=True)
+                    'mv {0} {1}'.format(full_path, new_path),  use_sudo=True)
```

Comment:
Extra space would fail flake8 if it ran on this repo I think

Reply:
Great catch